drivers/storage/ide/pciidex/power.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         PCI IDE bus driver extension
+ * PURPOSE:         Power support functions
+ * PROGRAMMERS:     Dmitry Borisov (di.sean@protonmail.com)
+ */
```

Comment:
Please use the new file headers format for the new files (or the files that have been heavily touched/rewritten).